src/main/resources/com/sumologic/sumobot/http_frontend/script.js

```diff
@@ -18,23 +18,39 @@
  */
 window.onload = function() {
     var websocketProtocol = window.location.protocol === "https:" ? "wss://" : "ws://";
-
     var endpoint = websocketProtocol + window.location.host + window.location.pathname + "websocket";
-
     var socket = new WebSocket(endpoint);
-    var messageBox = document.getElementById("messages");
 
+    var messageBox = document.getElementById("messages");
+    var messageInput = document.getElementById("message-input");
     var submitButton = document.getElementById("submit");
+
+    function appendMessage(msg, styles) {
+        var messageItem = document.createElement("div");
+        messageItem.textContent = msg;
+        messageItem.setAttribute("class", styles);
```

Comment:
Usually it's better to write to properties, though here it's mostly a code style issue:
```suggestion
        messageItem.className = styles;
```
If you switch to `classList` this code will change anyway, though.